components/collection/unlockable/UnlockableContainer.vue

```diff
@@ -170,11 +171,16 @@ import { timeAgo } from '@/components/collection/utils/timeAgo'
 import { collectionId, countDownTime } from './const'
 import { UNLOCKABLE_CAMPAIGN, createUnlockableMetadata } from './utils'
 import { endOfHour, startOfHour } from 'date-fns'
+import type Vue from 'vue'
+import { ConnectWalletModalConfig } from '@/components/common/ConnectWallet/useConnectWallet'
+
 const { toast } = useToast()
 
 const Loader = defineAsyncComponent(
   () => import('@/components/shared/Loader.vue')
 )
+const { $buefy } = useNuxtApp()
```

Comment:
I would like to see something else, please consider refactoring ref #5852

Reply:
I guess buefy modal component should be refactored on issue #5852 . For now, we have to use it to open the wallet modal.